node/src/main/scala/coop/rchain/node/NodeRuntime.scala

```diff
@@ -77,14 +88,19 @@ class NodeRuntime private[node] (
   private val storageSize       = conf.server.mapSize
   private val defaultTimeout    = conf.server.defaultTimeout // TODO remove
 
+  implicit class RichTask[A](t: Task[A]) {
```

Comment:
I've seen such utility classes named `*Ops` in other projects, so could you consider renaming to `TaskOps`?